packages/models-library/src/models_library/projects_nodes.py

```diff
@@ -26,30 +28,34 @@
     PortLink,
     SimCoreFileLink,
 )
-from .boot_options import EnvVarKey
 from .projects_nodes_ui import Position
 from .projects_state import RunningState
 from .services import PROPERTY_KEY_RE, SERVICE_KEY_RE
 
 # NOTE: WARNING the order here matters
+
 InputTypes = Union[
     StrictBool,
     StrictInt,
     StrictFloat,
+    Json,  # FIXME: remove if OM sends object/array. create project does NOT use pydantic
```

Comment:
This is one of the drawback of our "one model for all" :-(
The ``Json`` type is used in the ``ProjectPut`` model so that the front-end just sends a string, the model converts to dict/list and stores it that way in the database.